BusinessTampereTrafficMonitoring/tools/lane_configurator.py

```diff
@@ -73,6 +83,9 @@ def main():
     infotxt = tk.StringVar()
 
     def update_infotxt():
+        """
+        called from main method. Helps in selecting points in the input image
```

Comment:
This is not what this function does. This function updates the info text.

More importantly, there is no need to have docstrings for closures (which are functions defined inside functions), because they can't even be seen from outside the outer function. They are an implementation detail and not part of the API.